scss/main.scss

```diff
@@ -0,0 +1,38 @@
+@import "variables";
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  padding: 71px 20px 0 20px;
+  display: flex;
+  flex-direction: column
```

Comment:
put a semicolon at the end of the line and, order alphabetically

Reply:
e2be82f